go.mod

```diff
@@ -3,44 +3,46 @@ module github.com/openstack-exporter/openstack-exporter
 go 1.22
 
 require (
-	github.com/alecthomas/kingpin/v2 v2.3.2
+	github.com/alecthomas/kingpin/v2 v2.4.0
```

Comment:
These are the upgrades used inhttps://github.com/openstack-exporter/openstack-exporter/pull/393

I mostly did that so I could find a Prometheus version that passes unit tests. It seems that above these versions things with timestamps start breaking.